core/src/test/scala/za/co/absa/fadb/statushandling/fadbstandard/StandardStatusHandlingSuite.scala

```diff
@@ -17,30 +17,32 @@
 package za.co.absa.fadb.statushandling.fadbstandard
 
 import org.scalatest.funsuite.AnyFunSuite
-import za.co.absa.fadb.statushandling.StatusException
+import za.co.absa.fadb.naming_conventions.{NamingConvention, SnakeCaseNaming}
+import za.co.absa.fadb.statushandling.{FunctionStatus, StatusException}
 import za.co.absa.fadb.statushandling.StatusException._
 
 import scala.reflect.ClassTag
-import scala.util.Try
+import scala.util.{Success, Try}
 
-class StandardStatusHandlingTest extends AnyFunSuite {
+class StandardStatusHandlingSuite extends AnyFunSuite {
   test("Verify checkStatus error mapping") {
     class StandardStatusHandlingForTest extends StandardStatusHandling {
-      override def checkStatus(status: Integer, statusText: String): Try[Unit] = super.checkStatus(status, statusText)
+      override def checkStatus(status: FunctionStatus): Try[FunctionStatus] = super.checkStatus(status)
+      override def checkStatus(status: Integer, statusText: String): Try[FunctionStatus] = super.checkStatus(status, statusText)
       override def functionName: String = "Never needed"
+      override def namingConvention: NamingConvention = SnakeCaseNaming.Implicits.namingConvention
     }
 
     def assertCheckStatusFailure[F <: StatusException](status: Int, statusText: String)
                                                       (implicit classTag: ClassTag[F], checker: StandardStatusHandlingForTest): Unit = {
       val failure = intercept[F] {
         checker.checkStatus(status, statusText).get
       }
-      assert(failure.status == status)
-      assert(failure.statusText == statusText)
+      assert(failure.status == FunctionStatus(status, statusText))
     }
     implicit val standardStatusHandling: StandardStatusHandlingForTest = new StandardStatusHandlingForTest
 
-    assert(standardStatusHandling.checkStatus(10, "OK").isSuccess)
+    assert(standardStatusHandling.checkStatus(FunctionStatus(10, "OK")) == Success(FunctionStatus(10, "OK")))
```

Comment:
What about the last one? Perhaps you could also use `assertCheckStatusFailure` instead of assert, like this: `assertCheckStatusFailure[DBFailException]` ❓

Reply:
Will do with the comparison function.